pyproject.toml

```diff
@@ -126,6 +126,7 @@ dev = [
     "tokenizers",
     "aiohttp",
     "beautifulsoup4",
+    "uv",
```

Comment:
At this point, we haven't fully committed to `uv` other than in CI, which is why we have `ci-*` variants of make targets. I also think we'll generally lean toward `uv` as a system dependency rather than managed through `pip`, but that could change in the future.

For the time being, though, we're intentionally trying to avoid `uv` being something that a contributor would need to install.

Reply:
ok, I'll drop this!